tools/cli/adminCommands.go

```diff
@@ -62,3 +65,48 @@ func AdminDescribeWorkflow(c *cli.Context) {
 
 	prettyPrintJSONObject(resp)
 }
+
+// AdminDescribeHistoryHost describes history host
+func AdminDescribeHistoryHost(c *cli.Context) {
+	// using service client instead of cadence.Client because we need to directly pass the json blob as input.
+	serviceClient := getAdminServiceClient(c)
+
+	wid := c.String(FlagWorkflowID)
+	sid := c.String(FlagShardID)
+	addr := c.String(FlagAddress)
+	printFully := c.Bool(FlagPrintFullyDetail)
+
+	if len(wid) <= 0 && len(sid) <= 0 && len(addr) <= 0 {
+		ExitIfError(fmt.Errorf("at least one of them is required to provide to lookup host: workflowID, shardID and host address"))
```

Comment:
Let's deprecated ExitIfError and use ErrorAndExit. The former one had a confusing STACK_TRACE thing and lack of color and human readable print.

Reply:
Sounds good.